account_multicurrency_revaluation/__manifest__.py

```diff
@@ -10,6 +10,7 @@
  "license": 'AGPL-3',
  "depends": [
      "account_reversal",
+     "currency_monthly_rate",
```

Comment:
:warning: We should not **force** users already using the multicurrency revaluation module to install `currency_monthly_rate` !

IMO it would be cleaner to create a new 'glue' module depending on both `account_multicurrency_revaluation` and `currency_monthly_rate` for this usage.

Reply:
I agree with @grindtildeath